sql/ddl_internal.sql

```diff
@@ -185,23 +185,15 @@ DECLARE
 BEGIN
     SELECT * INTO STRICT trigger_row FROM pg_trigger WHERE OID = trigger_oid;
 
-    IF trigger_row.tgname = ANY(_timescaledb_internal.timescale_trigger_names()) THEN
-        RETURN FALSE;
-    END IF;
-
     IF (trigger_row.tgtype & (1 << 2) != 0) THEN
         --is INSERT trigger
-        IF (trigger_row.tgtype & (1 << 3) != 0) OR  (trigger_row.tgtype & (1 << 4) != 0) THEN
+        IF (trigger_row.tgtype & (1 << 3) != 0) OR (trigger_row.tgtype & (1 << 4) != 0) THEN
```

Comment:
Yes pushed changes and test to mat/chunk-dispatch (since I couldn't push to your branch)